android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/RepeatingGroupFactory.java

```diff
@@ -377,23 +336,10 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
             @Override
             public void afterTextChanged(Editable s) {
                 doneButton.setImageResource(R.drawable.ic_done_grey);
-                ValidationStatus validationStatus = JsonFormFragmentPresenter
-                        .validate(widgetArgs.getFormFragment(), referenceEditText, false);
-                if (validationStatus.isValid()) {
-                    writeJsonObjectValue(repeatingGroupCount, s.toString());
-                }
```

Comment:
I see the logic to write into the count object was moved but is the validation still being done? i.e. is `JsonFormFragmentPresenter
                        .validate(widgetArgs.getFormFragment(), referenceEditText, false);` still called?

Reply:
It's useful where the reference edit text has validators added.

Reply:
good catch